python/pickle/PyROOT_writetests.py

```diff
@@ -75,7 +75,14 @@ def test4WriteROOTObjInNamespace( self ):
    def test5WriteCustomTypes( self ):
       """Test writing PyROOT custom types"""
 
-      o = [ ROOT.Long(123), ROOT.Double(123.123) ]
+      self.exp_pyroot = os.environ.get('EXP_PYROOT') == 'True'
+
+      if self.exp_pyroot:
+         import ctypes
```

Comment:
Same comment as above, since it is a different test it would be good to have the clarification of why we test `ctypes` also here.

Reply:
Same comment as above here.